### PR TITLE
[jsk_pcl_ros] Fixed mistake of condition in edge_depth_refinment

### PR DESCRIPTION
**Pull request by iory**
condition max_x_index != max_x_index is always false

### CODE REVIEW

File: jsk_pcl_ros/src/edge_depth_refinement_nodelet.cpp [hunk 1]
```diff
@@ -143,7 +143,7 @@ namespace jsk_pcl_ros
       }
     }
 
-    if (max_x_index != max_x_index) {
+    if (min_x_index != max_x_index) {
       return boost::make_tuple(
         max_x_index, min_x_index);
```
Comment (wkentaro) [inline jsk_pcl_ros/src/edge_depth_refinement_nodelet.cpp]: Why is it in this order?
Here it is `max, min`, but `min, max` [below](https://github.com/jsk-ros-pkg/jsk_recognition/pull/1720/files#diff-5475a8fb677341120b99f38181eb936aR164).

File: jsk_pcl_ros/src/edge_depth_refinement_nodelet.cpp [hunk 2]
```diff
@@ -143,7 +143,7 @@ namespace jsk_pcl_ros
       }
```
Comment (wkentaro) [inline jsk_pcl_ros/src/edge_depth_refinement_nodelet.cpp]:
```c++
      if (y > max_y) {
         // max_y = max_y;
         max_y = y;
         max_y_index = index;
       }
```

Comment (iory) [inline jsk_pcl_ros/src/edge_depth_refinement_nodelet.cpp]: I overlooked this

**Comment by wkentaro** [comment]
IMO, we should add a simple test if we change the existing program.

**Comment by iory** [comment]
Ok, I will write test code.

**Comment by iory** [comment]
Please merge this.

**Pull request merged**